File: sentimental-cash/cash.py
from cs50 import get_float

while True:
    dollar = get_float("Change:")
    if dollar >= 0:
        break

if dollar == 0:
    print(0)

change = int(round(dollar*100))

i = 0

while change >= 25:
    i += 1
    change -= 25

while change >= 10:
    i += 1
    change -= 10

while change >= 5:
    i += 1
    change -= 5
while change >= 1:
    i += 1
    change -= 1
print(i)

File: cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    int change=get_int("Change owed=");
    int i=0;
    if(change>0)
    {
        while(change-25>=0)
        {
            i=i+1;
            change=change-25;
        }
        while(change-10>=0){
            i=i+1;
            change=change-10;
        }
        while(change-5>=0){
            i=i+1;
            change=change-5;
        }
        while(change-1>=0){
            i=i+1;
            change=change-1;
        }
        printf("%i\n",i);
    }
    else if(change==0)
    {
        printf("0\n");
    }
}
